src/calendars/czechRepublic.ts

```diff
@@ -202,6 +202,7 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
     },
     {
       key: 'lawrence_ruiz_and_companions_martyrs',
+      date: dayjs.utc(`${year}-9-28`),
```

Comment:
AFAIK when dropping a key, romcal does not require defining a date, does it?

Reply:
Actually it's required. But I agree that it makes no sense. 

I'll try to address that in this PR.